satpy/modifiers/parallax.py

```diff
@@ -0,0 +1,474 @@
+# Copyright (c) 2021-2022 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Parallax correction.
+
+Routines related to parallax correction using datasets involving height, such
+as cloud top height.
+
+The geolocation of (geostationary) satellite imagery is calculated by
+agencies or in satpy readers with the assumption of a clear view from
+the satellite to the geoid.  When a cloud blocks the view of the Earth
+surface on the surface is above sea level, the geolocation is not accurate
```

Comment:
```suggestion
surface and the surface is above sea level, the geolocation is not accurate
```

I think `and` was intended?

Reply:
Actually, I meant or.  Correcting mountains is another possible application for parallax correction.

Reply:
Fixed in d8749946.